pkgs/applications/blockchains/chia/default.nix

```diff
@@ -16,14 +16,14 @@ python3Packages.buildPythonApplication rec {
   };
 
   patches = [
-    # tweak version requirements to what's available in Nixpkgs
-    ./dependencies.patch
     # Allow later websockets release, https://github.com/Chia-Network/chia-blockchain/pull/6304
     (fetchpatch {
       name = "later-websockets.patch";
       url = "https://github.com/Chia-Network/chia-blockchain/commit/a188f161bf15a30e8e2efc5eec824e53e2a98a5b.patch";
       sha256 = "1s5qjhd4kmi28z6ni7pc5n09czxvh8qnbwmnqsmms7cpw700g78s";
     })
+    # tweak version requirements to what's available in Nixpkgs
+    ./dependencies.patch
```

Comment:
Why not use substituteInPlace?

Reply:
That'd be just as fragile, wouldn't it?

Reply:
It is actually less fragile because it does not break when surrounding lines change. Also it keep the repo smaller.

Reply:
Indeed